modules/common/src/main/resources/reference.conf

```diff
@@ -280,3 +280,17 @@ surge {
   dr-standby-enabled = false
   dr-standby-enabled = ${?SURGE_ENABLE_DR_STANDBY}
 }
+
+prober {
+    initial-delay = 3 seconds
+    timeout = 50 milliseconds
+    interval = 5 seconds
+    num-probes = 7
+    dispatcher {
```

Comment:
Yeah sorry mea culpa ... The prober should run on its own dispatcher (not only in tests but also in prod) so it can be immune from thread starvation issues ... So, it needs to sit 'outside' so it can examine what's going on with the other execution contexts , let me know if it makes sense ;-)